docs/tutorials/md_tutorial.nblink

```diff
@@ -0,0 +1,3 @@
+{
+   "path": "../ExampleNotebooks/openmm_md/plain_md.ipynb",
```

Comment:
The docs build issue is unclear, but I _think_ it might be that it doesn't like the errand `,` at the end?

```suggestion
   "path": "../ExampleNotebooks/openmm_md/plain_md.ipynb"
```

Reply:
I changed that but docs are still failing, I think because the PR in the Examplenotebook repo first needs to get approved!